finalProject.html

```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="finalProject.css">
     <title>Store</title>
+    <script src="store.js" async></script>
```

Comment:
this should be placed just before the </body> tag below. This helps make sure that you JS code can reference the code above when its loaded.

Reply:
Weird I swear I typed an example of the tag lol sorry, right before/above the closing body tag

Reply:
ah okay! Sounds good brother man thank ya!